src/Core/Framework/Adapter/Database/MySQLFactory.php

```diff
@@ -44,6 +44,16 @@ public static function create(array $middlewares = []): Connection
             ],
         ], $parameters); // adding parameters that are not in the DSN
 
+        if (EnvironmentHelper::getVariable('SQL_SET_DEFAULT_SESSION_VARIABLES', true)) {
```

Comment:
I'm not sure if with this change we would still need the env variable 🤔 
The way i understand it the variables are set during connecting and should not have a negative performance impact

@shyim what do you think? Can we remove that toggle completely?

Reply:
we also advise the people to set those variables directly in MySQL: https://developer.shopware.com/docs/guides/hosting/performance/performance-tweaks.html#mysql-configuration

It would be nice, if we enforce them directly, I guess :thinking: but I am not sure, if this would have performance impact during setting up the connection :thinking: I also could not find something related to this. 

maybe we can take some measurements?

Reply:
I can not find any documentation, that his could have any significant impact on performance during my research. 
I did some performance tests locally and could not see a performance impact either.

I think, it is safe to say, that the performance impact is negligible at max.
We are setting these variables once per connection and these specific commands do not seem to have any unreasonable impact, bar the other parameters we send to the DB anyways

Reply:
So therefore I proprose to completely remove the check of the `SQL_SET_DEFAULT_SESSION_VARIABLES` flag to ensure the perfomance tweaks are set per default in the session

Reply:
agree, when it pops up again we can put a feature flag behind it